.github/workflows/release.yaml

```
@@ -118,7 +115,7 @@ jobs:
           else
             bin="target/${{ matrix.job.target }}/release/xh"
           fi
-          staging="xh-${{ steps.get_version.outputs.version }}-${{ matrix.job.target }}"
+          staging="xh-${GITHUB_REF_NAME}-${{ matrix.job.target }}"
```

Can we use `github.ref_name` here?
```suggestion
          staging="xh-${{ github.ref_name }}-${{ matrix.job.target }}"
```